packages/accounts-password/package.js

```diff
@@ -4,7 +4,7 @@ Package.describe({
 });
 
 Package.onUse(function(api) {
-  api.use('npm-bcrypt@0.8.7');
+  api.use('npm-bcrypt@=0.8.7');
```

Comment:
@markoshust I'm hoping this `@=` will help force Meteor 1.4 to use only 0.8.7.

https://github.com/meteor/meteor/issues/7339#issuecomment-233437250